### PR TITLE
Fixes https://github.com/qooxdoo/qooxdoo-compiler/issues/309

### PR DESCRIPTION
**Pull request by johnspackman**

### CODE REVIEW

File: test/testapp/qx-lock.json [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+      "path": "qx_packages/qooxdoo_qxl_testtapper_v0_3_1",
+      "uri": "qooxdoo/qxl.testtapper",
+      "repo_name": "qooxdoo/qxl.testtapper",
+      "repo_tag": "v0.3.1"
     }
-  ]
-}
+  ],
+  "version": "2.1",
+  "requires": {
```
Comment (cboulanger) [inline test/testapp/qx-lock.json]: This is wrong here-should have gone into the manifest....

Comment (johnspackman) [reply test/testapp/qx-lock.json]: weird - I didn't edit that file, just ran `qx migrate`

**Comment by johnspackman** [comment]
@cboulanger @hkollmann why has this suddenly failed with lint errors?  Have the lint rules been changed?

**Comment by hkollmann** [comment]
Rules have'nt change. We just check lint now.

**Comment by cboulanger** [comment]
I think we should disable the no-todo rule. I think it makes sense to not allow FIXMEs, but TODOs should be fine.

**Comment by hkollmann** [comment]
no-todo is just a warning. not an error.

**Comment by cboulanger** [comment]
@hkollmann Ah ok, thanks!

**Comment by johnspackman** [comment]
> no-todo is just a warning

yet another example of why lint is 99% useless.  Apparently, having a blank line before `} else if (...)` is  an error too.

> Strange fail in the test, Ill restart.

The fail is because the test is broken by this commit, fixing that now.

**Comment by cboulanger** [comment]
@johnspackman We can disable individual rules if we don't find them useful. In all, I find eslint really helpful to spot problems. If we really need all those strict curly braces rules, I don't know.

**Pull request merged**